spec/suites/map/handler/Map.DragSpec.js

```diff
@@ -142,7 +142,7 @@ describe("Map.Drag", function () {
 				onStop: function () {
 					expect(map.getZoom()).to.be(1);
 					// Expect center point to be the same as before the click
-					expect(map.getCenter()).to.be(originalCenter);
+					expect(map.getCenter()).to.eql(originalCenter);
```

Comment:
I would replace `map.setView(originalCenter.clone(), 1);` with `map.setView(originalCenter, 1);` and then add following test:
```
expect(map.getCenter()).to.eql(originalCenter);
expect(map.getCenter()).to.not.be(originalCenter);
```

Meaning: Same data but not the same object, because we break the reference with `clone()` in the backend


Maybe worth to add a own test instead of extending this one.

Reply:
The fact it's not the same object is tested elsewhere, in the added test (where `map.getCenter()` value is mutated).